vspd.go

```diff
@@ -74,6 +81,32 @@ func run(ctx context.Context) error {
 	}
 	defer db.Close()
 
+	// Check if admin password hash is set in database.
+	hash, err := db.GetAdminHash()
+	if cfg.AdminPass == "" && err != nil {
```

Comment:
Does this error always mean admin pass is not set? Could it be some other error?